roles/openshift_on_openstack_scaleup/vars/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+# A list of regular expressions to search and replace in openshift-ansible all.yml.
+all_yml:
+  - { find: "^openshift_openstack_num_masters.*", replace: "openshift_openstack_num_masters: 3" }
```

Comment:
A scale up operation only adds nodes. Why do you search and replace the number of masters here?

Reply:
Good point.  I wanted to have a generic scaleup and only really tested scaleup of worker nodes.  Never tested master scaleup.  Feel free to comment this out.